### PR TITLE
Migration script and connectivity test script

### PR DESCRIPTION
**Pull request by BrienHartung**
Added a draft of the migration script and a test for the scp and ssh functionality.

### CODE REVIEW

**Comment by BrienHartung** [comment]
Adding @dpeterka @vtrivedi for review.

The connectivity test script gets a "permission denied" ssh error when attempted from the old server. This is despite the fact that I can ssh into the new server from the old server.

File: scp-test.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# This should result in three text files in home directory, copies of same
+# in home directory of other server, and a text file with three lines in it 
+# in other server.
+ 
+USERLIST="alice bob charlie"
+
+for USER in $USERLIST
+do
+  echo "attempting $USER"
+  FILENAME=""$USER".txt"
```
Comment (dpeterka) [inline scp-test.sh]: FILENAME="$USER.txt"

File: scp-test.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  echo "This is a test for $USER" > $FILENAME
```
Comment (dpeterka) [inline scp-test.sh]: LOG=foo.txt
exec 1>l$LOG2>&1
#everything after gets written to $LOG
do stuff

Comment (BrienHartung) [reply scp-test.sh]: What does the "l" (lowercase L) in "exec 1>l$LOG2>&1" mean?

**Comment by BrienHartung** [comment]
I've split the commands that need to be done on each server into two separate scripts. Just tried it and it completes the migration process - still need to test the new sites for any issues.

**Pull request merged**